benchmark/profile_generation.py

```diff
@@ -273,66 +278,96 @@ class ProfileResult:
     parser.add_argument(
+        '-pt',
         '--prompt-tokens',
         nargs='+',
         type=int,
-        help='how many requests launched concurrently. One-to-one'
+        help='how many requests launched concurrently. One-to-one '
         'correspondence with completion-tokens',
         default=[1, 128, 128, 2048, 2048])
-    parser.add_argument('--completion-tokens',
+    parser.add_argument('-ct',
+                        '--completion-tokens',
                         nargs='+',
                         type=int,
                         help='how many tokens to be generated. One-to-one'
                         'correspondence with prompt-tokens',
                         default=[128, 128, 2048, 128, 2048])
-    parser.add_argument('--tp', type=int, help='Tensor parallel', default=1)
-    parser.add_argument('--top_k',
-                        type=int,
-                        help='The number of highest probability vocabulary '
-                        'tokens to keep for top-k-filtering',
-                        default=1)
-    parser.add_argument('--top_p',
-                        type=float,
-                        help='the set of most probable tokens with '
-                        'probabilities that add up to top_p or higher '
-                        'are kept for generation',
-                        default=1.0)
-    parser.add_argument('--temperature',
-                        type=float,
-                        help='The value used to modulate the next token '
-                        'probabilities',
-                        default=1.0)
     parser.add_argument('--csv',
                         type=str,
                         help='Where to save the result.',
                         default='profile_generation.csv')
-    parser.add_argument('--log-level',
-                        help='set log level',
-                        default='ERROR',
-                        choices=list(logging._nameToLevel.keys()))
-    parser.add_argument('--test-round',
+    parser.add_argument('-tr',
+                        '--test-round',
                         type=int,
                         help='number of test rounds',
-                        default=6)
-    parser.add_argument('--warmup-round',
+                        default=3)
+    parser.add_argument('-w',
+                        '--warmup-round',
                         type=int,
-                        help='number of warmuop rounds',
+                        help='number of warmup rounds',
                         default=1)
+    # other args
+    ArgumentHelper.top_p(parser)
+    ArgumentHelper.temperature(parser)
+    ArgumentHelper.top_k(parser)
+    ArgumentHelper.log_level(parser)
+    ArgumentHelper.backend(parser)
+    # pytorch engine args
+    pt_group = parser.add_argument_group('PyTorch engine arguments')
+    tp_act = ArgumentHelper.tp(pt_group)
+    session_len_act = ArgumentHelper.session_len(pt_group, default=4096)
```

Comment:
`python benchmark/profile_generation.py workspace/llama-2-7b-chat --concurrency 1 16 32 64 --csv /home/zhulin1/press_record/llama-2-7b-chat_20240219/generation.csv` 

error occurs: 

/home/zhulin1/miniconda3/envs/lmdeployv23/lib/python3.10/site-packages/fuzzywuzzy/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
Traceback (most recent call last):
  File "/home/zhulin1/lmdeployLvhan/press_llama2/../benchmark/profile_generation.py", line 454, in <module>
    main()
  File "/home/zhulin1/lmdeployLvhan/press_llama2/../benchmark/profile_generation.py", line 372, in main
    args = parse_args()
  File "/home/zhulin1/lmdeployLvhan/press_llama2/../benchmark/profile_generation.py", line 333, in parse_args
    session_len_act = ArgumentHelper.session_len(pt_group, default=4096)
TypeError: ArgumentHelper.session_len() got an unexpected keyword argument 'default'

Reply:
Is `workspace/llama-2-7b-chat` in turbomind model format?

Reply:
@zhulinJulia24 hi, could you check if you are using the latest code? The method `session_len` should have `default` argument as in here
https://github.com/InternLM/lmdeploy/blob/da0d3eb6b5d2e38ed7130b6f4c8b17f8d21910b5/lmdeploy/cli/utils.py#L110

Reply:
> python benchmark/profile_generation.py workspace/llama-2-7b-chat --concurrency 1 16 32 64

The error occurs when only code is update. 
I try to build a new package, it works fine.